x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/FieldAttributeTests.java

```diff
@@ -349,6 +351,45 @@ public void testUnsignedLongVersionCompatibility() {
         }
     }
 
+    public void testVersionFieldVersionCompatibility() {
```

Comment:
`testVersionTypeVersionCompatibilty()`? `SELECT '1.2.3'::version'...` makes use of no field to test.